package.json

```diff
@@ -5,6 +5,7 @@
     "@rails/webpacker": "rails/webpacker",
```

Comment:
You need rails/webpacker to be at the same version.

Reply:
as the gem

The change made was in the JS files, so your change here to the gem only has no effect